styles.css

```diff
@@ -0,0 +1,93 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --font-family-body: 'Poppins', sans-serif;
+  --ff-headings: 'Playfair Display', serif;
+  --font-size-h1: clamp(1.5rem, -0.25rem + 20vw, 6rem);
+  --gap-header: clamp(12rem, 100vw, 25rem);
+  --button-SC-width: min(90vw, 700px);
+  --adaptive-sizing-section-2: min(100vw, 700px);
+  --fs-body: clamp(1rem, 0.3846153846153846rem + 3.076923076923077vw, 1.5rem);
+}
+
+header {
+  display: flex;
+  gap: 37vw;
+  margin-bottom: 2rem;
+  height: 20vh;
+}
+
+.section-1 {
+  position: relative;
+}
```

Comment:
- [ ] The senior code reviewer mentioned this again and strongly recommended you change it, according to the requirement, you should use `flex` to align your headline section(means this one`(section-1)`) but in your case you have not yet done that, it is quick and easy to add `display: flex` before the line 25, see? why don't you fix it and come back with a ready-to-approve version of your project? am sure you just forgot to add it, so please as I reminded you, make sure you have this issue fixed and resubmit your work, other issues are well addressed already.